sp800-63b/sec5_authenticators.md

```diff
@@ -335,7 +335,7 @@ When the subscriber successfully authenticates, the verifier SHOULD disregard an
 
 For a variety of reasons, this document supports only limited use of biometrics for authentication. These include:
 
-- Biometric False Match Rates (FMR) and False Non-Match Rates (FNMR) do not provide confidence in the authentication of the subscriber by themselves. In addition, FMR and FNMR do not account for spoofing attacks.
+- FMR and FNMR do not provide confidence in the authentication of the subscriber by themselves. In addition, FMR and FNMR do not account for spoofing attacks.
```

Comment:
The abbreviations FMR and FNMR are only introduced in the definition of EER in Section 3. Please leave the words.